examples/fluent-bit/amazon-opensearch-serverless/README.md

```diff
@@ -0,0 +1,34 @@
+### FireLens Example: Logging to Amazon OpenSearch Serverless with Fluent Bit
+
+Amazon OpenSearch Serverless is new offering that eliminates your need to manage OpenSearch clusters. All existing Fluent Bit OpenSearch output plugin options work with OpenSearch Serverless. The only difference with serverless from a Fluent Bit POV is that you must specify the service name as `aoss` (Amazon OpenSearch Serverless) when you enable `AWS_Auth`:
+
+```
+AWS_Auth On
+AWS_Region <aws-region>
+AWS_Service_Name aoss
+```
+
+### Data Access Permissions
+When sending logs to OpenSearch Serverless, your task role (e.g. `arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_iam_role`) needs [OpenSearch Serverless Data Access permisions](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-data-access.html). Give your task role the following Data Access permissions to your serverless collection:
+
+```
+aoss:CreateIndex
+aoss:UpdateIndex
+aoss:WriteDocument
+```
+No task role IAM policies are needed to access the collection.
+
+### Adding Permissions with AWS CLI
+To add Data Access permissions to your task role via AWS CLI, use the following command along with the `aoss-data-access-policy.json` file from the `amazon-opensearch-serverless` example folder. Be sure to update the `aoss-data-access-policy.json` document with your collection name, and task role arn.
+```
+aws opensearchserverless create-access-policy \
+    --name log-write-policy \
+    --type data \
+    --policy  file://./aoss-data-access-policy.json
+```
+ Please note that the `opensearchserverless` command was introduced in aws cli version 2.x.x. If you recieve the error: `argument command: Invalid choice`, please [update your AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
```

Comment:
Let's not put "2.x.x" in our docs is possible. I bet you can find this in AWS CLI release notes: https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst#L383

I see it there for 1.x

Reply:
Updated